02/PrintRecord.cpp

```diff
@@ -0,0 +1,69 @@
+#include <cassert>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include "PrintRecord.h"
+
+using namespace std;
+
+namespace samples
+{
+	void PrintRecordExample()
+	{
+		const int NAME_SIZE = 10;
+		size_t count = 0;
+		string record;
+
+		string firstName;
+		string lastName;
+		string  studentID;
+		unsigned int score;
+
+		while (count < 4)
```

Comment:
just talked to Rachel..  let's remove cin part..

start from a fixed string

"Pope Kim A12345678 18"

feed that to inputstringstream...